src/etc/rc.newipsecdns

```diff
@@ -36,21 +49,20 @@ require_once("auth.inc");
 require_once("ipsec.inc");
 require_once("vpn.inc");
 
-/* make sure to wait until the boot scripts have finished */
-if (file_exists("{$g['varrun_path']}/booting")) {
-	return;
-}
-
-if (ipsec_enabled()) {
-	sleep(15);
-	log_error("IPSEC: One or more IPsec tunnel endpoints has changed its IP. Refreshing.");
-} else {
+if (!ipsec_enabled()) {
+	unlock($ipseclck_pending);
```

Comment:
IMO it would be better to check earlier if IPsec is enabled before try_lock() call and if it's not it doesn't need to try to lock